src/globalFilters/FilterField.vue

```diff
@@ -10,8 +10,8 @@
             name="setSelectionThreshold"
             @change="updateFilterValueFromDropdown($event, filter.comparator, $event.target.value)"
           >
-            <option key="NONE" value="NONE">
-              None
+           <option value="NONE">
+            {{ defaultLabel }}
```

Comment:
I think its great to allow devs to create a default label. Are end users going to be confused though by a select option in the dropdown that doesn't actually filter?

Reply:
explained off-line that it is a more descriptive way of telling the end-user what they are filtering on, even if it isn't anything, aka, "filtering" on A and B side means not filtering.